internal/credit/creditdriver/grant.go

```diff
@@ -34,7 +34,34 @@ func (b *builder) ListLedgerGrants() ListLedgerGrantsHandler {
 			}
 			return request, nil
 		},
-		b.CreditConnector.ListGrants,
+		func(ctx context.Context, request credit.ListGrantsParams) ([]api.LedgerGrantWithLedgerIDResponse, error) {
+			grants, err := b.CreditConnector.ListGrants(ctx, request)
+			if err != nil {
+				return nil, err
+			}
+			resp := make([]api.LedgerGrantWithLedgerIDResponse, 0, len(grants))
+			for _, grant := range grants {
+				mapped := mapGrantToAPI(grant)
+				resp = append(resp, api.LedgerGrantWithLedgerIDResponse{
```

Comment:
I understand that this is only used here, but it might worth placing it closer to the mapGrantToAPI, so that at least people updating the grant API don't forget to update the ledger version too.

Reply:
moved it next to it